src/unix/syscall.c

```diff
@@ -1931,6 +2025,8 @@ sysreturn sched_getaffinity(int pid, u64 cpusetsize, cpu_set_t *mask)
 sysreturn capget(cap_user_header_t hdrp, cap_user_data_t datap)
 {
     if (datap) {
+        if (!validate_user_memory(datap, sizeof(cap_user_data_t), true))
```

Comment:
cap_user_data_t is a pointer type, we should use either `struct user_cap_data` or `sizeof(*datap)`.